src/components/AddProject/AddProject.js

```diff
@@ -51,16 +51,21 @@ export default class AddProject extends React.Component {
       };
     }
     axios
-      .get(
+      .get( 
         `https://api.github.com/repos/${this.state.input}/commits/master/status`,
```

Comment:
This is still making api calls using the master :ref instead of latest commit ref

Reply:
But then will user have to enter commit id as well as part of input

Reply:
This code is making call using master ref to fetch latest commit ref API URL from response and getting status from that only

Reply:
It's a nested axios call

Reply:
if https://api.github.com/repos/cucumber/cucumber-jvm/commits/master/status returns a 404 then grab the sha from https://api.github.com/repos/cucumber/cucumber-jvm/commits?per_page=1 and recall https://api.github.com/repos/cucumber/cucumber-jvm/commits/:sha:/status